apps/core/app/[locale]/(default)/account/[tab]/page.tsx

```diff
@@ -0,0 +1,61 @@
+import type { Metadata } from 'next';
+import { notFound } from 'next/navigation';
+import { getTranslations } from 'next-intl/server';
+
+import { LocaleType } from '~/i18n';
+
+export type TabType =
+  | 'orders'
+  | 'messages'
+  | 'addresses'
+  | 'wishlists'
+  | 'recently-viewed'
+  | 'settings';
```

Comment:
Maybe we can make this a single source of truth?
```suggestion

export const tabList = ['orders', 'messages', 'addresses', 'wishlists', 'recently-viewed', 'settings'] as const
export type TabType = keyof tabList;
```

Reply:
Thanks, Chance. I've updated but with small changes. while trying to export `tabList` I got an error `... is not a valid Next.js entry export value` and since we use `tablist` as prop in `<AccountTabs/>` so I moved both of them to layout and export  only `TabType` across page & component. What do you think?